src/sentry/tasks/post_process.py

```diff
@@ -1563,6 +1563,20 @@ def kick_off_seer_automation(job: PostProcessJob) -> None:
     event = job["event"]
     group = event.group
 
+    # check currently supported issue categories for Seer
+    if group.issue_category not in [
+        GroupCategory.ERROR,
+        GroupCategory.PERFORMANCE,
+        GroupCategory.MOBILE,
+        GroupCategory.FRONTEND,
+        GroupCategory.DB_QUERY,
+        GroupCategory.HTTP_CLIENT,
+    ] or group.issue_category in [
+        GroupCategory.REPLAY,
+        GroupCategory.FEEDBACK,
+    ]:
```

Comment:
don't we only need the first condition? isn't the or unnecessary?

Reply:
ah i am not sure about this since now we have v2. @malwilley, mind taking a look?

Reply:
I think this should work. Though I do want to note that FRONTEND also contains `performance_render_blocking_asset_span` and `performance_uncompressed_assets`. You can see which types map to what category in `issues/grouptype.py`